fastlane/Fastfile

```diff
@@ -37,6 +37,7 @@ platform :ios do
       configuration: "Debug",
       json_report: true,
       ignore_file_path: "fastlane/.xcovignore",
+      exclude_targets: "PayPalNativeCheckout"
```

Comment:
any idea how to ignore `PaypalClient.swift` from code coverage, decided to do it in different story because there's lot of mocking needed to cover this code
@kaylagalway @jaxdesmarais ^^

Reply:
We can remove this line and add the file path(s) here: https://github.com/paypal/iOS-SDK/blob/main/fastlane/.xcovignore

Reply:
Karthik can you explain more what you mean by removing this file from code coverage? Do you mean from linting?

Reply:
@scannillo  oh this shouldn't be there, I will take it off, this was to exclude few files since we couldn't figure out how to mock them but that's done in below `xcovignore`

Reply:
Ok - I think I don't understand why the file was adding to `xcovignore`?

Reply:
See https://github.com/paypal/iOS-SDK/pull/69/files#r953970884

Reply:
@kgangineni - we still need to remove this line